app/lib/screens/HomeScreens/chat/ChatProfile.dart

```diff
@@ -27,10 +27,19 @@ class ChatProfileScreen extends StatefulWidget {
 }
 
 class _ChatProfileScreenState extends State<ChatProfileScreen> {
-  String chatName = '';
+  String? chatName;
   String chatDesc =
       'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec aliquam ex. Nam bibendum scelerisque placerat.';
 
+  @override
+  void initState() {
+    super.initState();
+
+    widget.room.displayName().then((value) {
```

Comment:
This call should be wrapped with ```if(mounted)``` condition as a safe-check to avoid uncertain crash and actually checking first if widget exists in tree.

Reply:
Understood.
I replaced all occurrences.

Reply:
This is only required if you use ```setState(){}``` in ```initState(){}```. Not needed everywhere though :)